index.js

```diff
@@ -24,10 +24,17 @@ module.exports = function (val, opts, pad) {
 		}
 
 		if (Array.isArray(val)) {
+
```

Comment:
@melonmanchan Unnecessary new line.
You may want to install `xo` to help you with the existing codestyle: `npm install xo`